src/gsy_e/models/strategy/scm/pv.py

```diff
@@ -0,0 +1,54 @@
+from typing import Union, Dict
+
+from gsy_e.models.strategy.predefined_pv import (
+    PVPredefinedEnergyParameters, PVUserProfileEnergyParameters)
+from gsy_e.models.strategy.pv import PVEnergyParameters
+from gsy_e.models.strategy.scm import SCMStrategy
+
+
+class SCMPVStrategy(SCMStrategy):
+    """PV SCM strategy with gaussian power production."""
+
+    def __init__(self, panel_count: int = 1, capacity_kW: float = None):
```

Comment:
IRRC, panel_count will no longer be advertised. The produced energy of a PV should only be scaled with the `capacity_kW`. The deprecation of `panel_count` will probably a big change and does not need to be done soon, but IMO we could omit it here already. Or is this a requirement to have this parameter also in the SCM PV? What do you think?

Reply:
I deliberately left it here in case the business requirements change, however you are right, YAGNI. I will remove the panel_count from the constructor arguments, and hardcode the value in the energy_params to 1, thanks!